Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -778,9 +777,22 @@ private bool IsProjectGlobalDeclaration(Declaration candidateDeclaration, Declar
         {
             return candidateDeclaration.ProjectName == scopingDeclaration.ProjectName
                 && !(candidateDeclaration.ParentScopeDeclaration is ClassModuleDeclaration)
-                && (candidateDeclaration.Accessibility == Accessibility.Public
-                    || ((candidateDeclaration.Accessibility == Accessibility.Implicit)
-                        && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)));
+                && (IsExplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration)
+                || IsImplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration));
+        }
+
+        private bool IsExplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
+        {
+            return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
+                    && candidateDeclaration.Accessibility == Accessibility.Public;
+        }
+
+        private bool IsImplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
+        {
+            return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
+                        && candidateDeclaration.Accessibility == Accessibility.Implicit
+                        && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)
+                        && !candidateDeclaration.IdentifierName.StartsWith("Option ");
```

Comment:
Wow...not sure what to say other than sorry.  DeclarationType.ModuleOption works like one would  expect now.   No explanation why...but the string op is gone.